slack/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,38 @@
+= Slack Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Slack (Slack Connector) enables access to the Slack platform. This connector exposes all of the operations provided by Slack's REST API.
```

Comment:
"Anypoint Connector for Slack (Slack connector) give you access to the Slack platform. This connector exposes key operations provided by the Slack API. Please visit Slack's [official site](https://slack.com/help/articles/115004071768-What-is-Slack-) to learn more.